server/database/config/dbbuild.sql

```diff
@@ -0,0 +1,42 @@
+BEGIN;
+DROP TABLE IF EXISTS users, languages, channels, user_channel, user_interest, messages CASCADE;
+CREATE TABLE languages (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL
+);
```

Comment:
it's better to add a line between the table and the next table 
its make the file more readable